src/xcode/ENA/ENA/Source/Scenes/Events/Cells/EventTableViewCell.swift

```diff
@@ -44,6 +44,17 @@ class EventTableViewCell: UITableViewCell, ReuseIdentifierProviding {
 			containerView.backgroundColor = cellBackgroundColor
 		}
 	}
+	
+	override func setEditing(_ editing: Bool, animated: Bool) {
+		super.setEditing(editing, animated: animated)
+		if editing {
+			button.isEnabled = false
+			button.setTitleColor(.enaColor(for: .textPrimary2), for: .disabled)
+			button.layer.borderColor = .enaColor(for: .textPrimary2)
+		} else {
+			button.isEnabled = true
+		}
```

Comment:
Your code is completely fine! 
One simpler suggestion, if you want

```swift
if editing {
	button.setTitleColor(.enaColor(for: .textPrimary2), for: .disabled)
	button.layer.borderColor = .enaColor(for: .textPrimary2)
}
button.isEnabled = !editing
```

Reply:
Unfortunately, without else didn't work out 😔